worker/runc/runc.go

```diff
@@ -63,12 +57,16 @@ func NewWorkerOpt(root string, labels map[string]string, snapshotterName string)
 	}
 
 	mdb := ctdmetadata.NewDB(db, c, map[string]ctdsnapshot.Snapshotter{
-		snapshotterName: s,
+		snFactory.Name: s,
 	})
 	if err := mdb.Init(context.TODO()); err != nil {
 		return opt, err
 	}
 
+	// Create the image store.
+	imageStore := ctdmetadata.NewImageStore(mdb)
```

Comment:
This is unrelated to snapshotter itself but should be useful for dedupe jessfraz/img